### DISCUSSION TITLE
Does dubbo3 rest support?

### DISCUSSION THREAD

**Discussion by lakewatcher** [apache/skywalking#11243] [Q&A]
I invoke a dubbo3 rest service using apisix , but the traceids of the upstream and apisix are different

apisix (traceId) --http-->dubbo3 rest ( new traceId)

**Comment by wu-sheng** [comment]
https://skywalking.apache.org/docs/skywalking-java/v8.16.0/en/setup/service-agent/java-agent/supported-list/

From the docs, we don't have that support.
